pkg/api/validation/validation.go

```diff
@@ -2024,6 +2024,14 @@ func ValidateHostAliases(hostAliases []api.HostAlias, fldPath *field.Path) field
 	return allErrs
 }
 
+func ValidatePriorityName(priorityName string, fldPath *field.Path) field.ErrorList {
```

Comment:
A couple of comments about validation

(1) You should also do validation on Priority (not just PriorityName) even though it is set by an admission controller (user might write a buggy custom admission controller). BTW do we want to allow negative priorities?

(2) I think you need to modify ValidatePodUpdate() to make it possible to update PriorityName (and Priority)